extensions/amp-selector/0.1/amp-selector.js

```diff
@@ -319,6 +349,61 @@ export class AmpSelector extends AMP.BaseElement {
     }
   }
 
+  /**
+   * Handles toggle action.
+   * @param {number} index
+   * @param {boolean} value
+   */
+  toggle_(index, value) {
+    // Change the selection to the next element in the specified direction.
+    // The selection should loop around if the user attempts to go one
+    // past the beginning or end.
+    const selectedIndex_ = this.options_.indexOf(this.selectedOptions_[0]);
+
+    if (value == true) {
```

Comment:
nit: you can simplify this to `if (value) {`

Reply:
Changed the code.